docs/integrations/prefect-azure/aci_worker.md

```diff
@@ -0,0 +1,222 @@
+# Azure Container Instances Worker Guide
+
+## Why use ACI for flow run execution?
+
+ACI (Azure Container Instances) is a fully managed compute platform that streamlines running your Prefect flows on scalable, on-demand infrastructure on Azure.
+
+## Prerequisites
+
+Before starting this guide, make sure you have:
+
+- An Azure account and user permissions for provisioning resource groups and container instances.
+- The `azure` CLI installed on your local machine. You can follow Microsoft's [installation guide](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli).
+- Docker installed on your local machine.
+
+
+## Step 1. Create a resource group
+
+Azure resource groups serve as containers for managing groupings of Azure resources.
+
+Replace `<resource-group-name>` with the name of your choosing, and `<location>` with a valid Azure location name, like `eastus`.
+
+```bash
+export RG_NAME=<resource-group-name> && \
+az group create --name $RG_NAME --location <location>
+```
+
+Throughout the rest of the guide, we'll need to refer to the **scope** of the created resource group, which is a string describing where the resource group lives in the hierarchy of your Azure account. To save the scope of your resource group as an environment variable, run the following command:
+
+```bash
+RG_SCOPE=$(az group show --name $RG_NAME --query id --output tsv)
+```
+
+You can check that the scope is correct before moving on by running `echo $RG_SCOPE` in your terminal. It should be formatted as follows:
+
+```
+/subscriptions/<subscription-id>/resourceGroups/<resource-group-name>
+```
+
+## Step 2. Prepare ACI permissions
+
+In order for the worker to create, monitor, and delete the other Container Instances in which flows will run, we'll need to create a **custom role** and an **identity**, and then affiliate that role to the identity with a **role assignment**. When we start our worker, we'll assign that identity to the Container Instance it's running in.
+
+### 1. Create a role
+
+The custom `Container Instances Contributor` role has all the permissions your worker will need to run flows in other Container Instances. Create it by running the following command:
```

Comment:
```suggestion
The custom `Container Instances Contributor` role has all the permissions your worker will need to run flows in other container instances. Create it by running the following command:
```

Reply:
I think this should be lower case - but maybe they always capitalize it, in which case, please disregard the suggested change.